tests/core/test_i64.in

```diff
@@ -0,0 +1,29 @@
+#include <stdio.h>
```

Comment:
Sounds like that's worth doing then.
For interface I'm thinking of modifying the in_out_tests method to check for a list of valid input extensions `['.c', '.cpp']`, not let the user specify the extension explicitly, and assert that exactly one of the possible input files exists. That way at call site we don't have to care about extensions, but at compile time we know what our file is.
And for output files, force them all to be `.out` because it doesn't have any semantic meaning either way.
And then batch-update all the current `.in` files into `.c` or `.cpp`s.
Seem reasonable?